extensions/functions_geometry.yaml

```diff
@@ -0,0 +1,29 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "st_point"
+    description: >
+      Return a point with the given x and y coordinate values.
+    impls:
+      - args:
```

Comment:
PostGIS has at least three functions that create points but they all return geometry.  Do we want to also return a geometry?  Do we want to support 3D and 4D points?

Reply:
Agreed, sticking to two 2D for now is fine.